app/controllers/authentications_controller.rb

```diff
@@ -35,7 +35,12 @@ def destroy
         # Bryan: TESTED
         flash[:alert] = 'Bad idea!'
       elsif @authentication.destroy
-        flash[:notice] = 'Successfully removed profile.'
+	user = User.find(current_user.id)
+	if user.update_attributes(github_profile_url: nil)
+          flash[:notice] = 'Successfully removed profile.'
+	else
+          flash[:notice] = 'Github profile url could not be removed.'
```

Comment:
removing the code as a risk? yes indeed - but can we create new tests that reach these locations?  You think that's tricky?